### PR TITLE
[4.0]fix error during a new category creation

### PR DESCRIPTION
**Pull request by bahl24**
Pull Request for Issue #24202  .

### Summary of Changes
Fixed xml file to match with the correct sql table structure.

### Testing Instructions
Content->Field Groups->New->Save & close

### Expected result
No error and field is saved.

### Actual result
Save failed with the following error: Incorrect datetime value: '' for column 'modified' at row 1

### CODE REVIEW

File: administrator/components/com_fields/forms/group.xml [hunk 1]
```diff
@@ -74,11 +74,12 @@
 
 		<field
 			name="modified"
-			type="text"
+			type="calender"
```
Comment (richard67) [inline administrator/components/com_fields/forms/group.xml:77]: `calender`? That looks German to me. Shouldn't it be `calendar`?

Comment (bahl24) [reply administrator/components/com_fields/forms/group.xml:77]: Thanks for pointing out

**Comment by SharkyKZ** [comment]
This isn't right. Default value should be set in the table class.

**Comment by infograf768** [comment]
please look at the xml in 3.9.x for the exact code for this field

**Comment by infograf768** [comment]
it was modified in 
https://github.com/joomla/joomla-cms/pull/16962

**Comment by infograf768** [comment]
field.xml may also need some attention. it should use calendar as type, etc . look at what we have in 3.9

**Comment by bahl24** [comment]
Ok @infograf768

**Comment by infograf768** [comment]
@bahl24 
Let me do this one. I found the issue for group: it is in the xml AND table

**Comment by infograf768** [comment]
Closing in favor of https://github.com/joomla/joomla-cms/pull/24234
Please test.

**Pull request closed without merging**